src/rules/curlyRule.ts

```diff
@@ -72,6 +72,58 @@ export class Rule extends Lint.Rules.AbstractRule {
         type: "functionality",
         typescriptOnly: false,
         hasFix: true,
+        codeExamples: [
```

Comment:
These code examples are going to end up taking a lot of space in the source files _(good!)_. What are your thoughts on starting a precedent of putting them in a `ruleName.examples.ts` file that just exports an object with examples?

Reply:
@JoshuaKGoldberg I had a similar concern myself. I think the separate file approach makes sense :+1: and will update the curly rule example.